lib/pf/api.pm

```diff
@@ -907,6 +908,35 @@ sub fingerbank_process : Public {
     return (pf::fingerbank::process($args));
 }
 
+=head2 fingerbank_update_upstream_db
+
+=cut
+
+sub fingerbank_update_upstream_db : Public {
+    my ( $class ) = @_;
+
+    my ( $status, $status_msg ) = fingerbank::DB::update_upstream;
+    pf::fingerbank::sync_upstream_db();
+
+    pf::config::util::pfmailer(( subject => 'Fingerbank - Update upstream DB status', message => $status_msg ));
+
+    return;
+}
+
+=head2 fingerbank_submit_unmatched
+
+=cut
+
+sub fingerbank_submit_unmatched : Public {
+    my ( $class ) = @_;
+
+    my ( $status, $status_msg ) = fingerbank::DB::submit_unknown;
+
+    pf::config::util::pfmailer(( subject => 'Fingerbank - Submit unknown/unmatched fingerprints status', message => $status_msg ));
+
+    return;
```

Comment:
I feel like I'm always saying things twice.

Not this time, guess which one it's related to